ui/job-view/headerbars/TiersMenu.jsx

```diff
@@ -10,15 +10,15 @@ export default function TiersMenu(props) {
 
   return (
     <span className="dropdown">
-      <span
+      <button
```

Comment:
I think if you use the react strap Button component you won't need to add the type attribute.

Reply:
Does it also include the keyboard handler and tab stop? A button element gives us all of that for free. And type="button" is totally correct here since it isn't part of a form.

Reply:
Sorry, I missed this question Marco. The react-strap Button component does contain the button element it just does so in a react-based component using Bootstrap's CSS, so `type=button` is added by default. We use this library for most of Treeherder and Perfherder.